core/server/master/src/main/java/alluxio/master/metastore/rocks/RocksInodeStore.java

```diff
@@ -319,14 +322,26 @@ private void initDb() throws RocksDBException {
       } catch (Throwable t) {
         LOG.error("Failed to close previous rocks database at {}", mDbPath, t);
       }
-      try {
-        FileUtils.deletePathRecursively(mDbPath);
```

Comment:
This didn't work before?

Reply:
Before we only deleted the db path on re-initialization. Now we delete it on the first initialization as well

Reply:
switched back to calling FileUtils.deletePathRecursively